requirements_test.txt

```diff
@@ -7,6 +7,7 @@ pytest-cov~=4.0.0
 pytest-mock~=3.10.0
 setuptools~=66.0.0
 tox~=3.28.0
+pandas~=1.3.5
```

Comment:
pandas already exists in our [requirements.txt](https://github.com/mtgjson/mtgjson/blob/master/requirements.txt#L9) file. This shouldn't be necessary

Reply:
zach hates pandas confirmed